Source/EventAnalysis/KTCollinearTrackClustering.hh

```diff
@@ -0,0 +1,127 @@
+/**
+ @file KTCollinearTrackClustering.hh
+ @brief Contains KTCollinearTrackClustering
+ @details Groups collinear tracks into one
+ @author: E. Zayas
+ @date: May 1, 2017
+ */
+
+#ifndef KTCOLLINEARTRACKCLUSTERING_HH_
+#define KTCOLLINEARTRACKCLUSTERING_HH_
+
+#include "KTPrimaryProcessor.hh"
+
+#include "KTSlot.hh"
+#include "KTData.hh"
+#include "KTMemberVariable.hh"
+#include "KTProcessedTrackData.hh"
+
+#include <algorithm>
```

Comment:
I see where std::vector is used, but not anything from `algorithm`, `set`, or `list`.  If they're used in the soruce file, more the include lines there.  Otherwise just remove them.